media-libs/libavif/libavif-0.10.1.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib gnome2-utils
+
+DESCRIPTION="Library for encoding and decoding .avif files"
+HOMEPAGE="https://github.com/AOMediaCodec/libavif"
+SRC_URI="https://github.com/AOMediaCodec/libavif/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+# See bug #822336 re subslot
+SLOT="0/${PV}"
```

Comment:
FYI libabigail doesn't report any function changes, so maybe updating subslot with each bump is actually harmful without any gain?

Reply:
If SONAME is changing every time, we still have to rebuild even if ABI is the same, as SONAME is how things indicate ABI compatibility (so they'll refuse to run and preserve-libs kicks in). May be worth speaking to upstream: might be that they preemptively do it because they often break ABI but not necessarily in every release but haven't set up some monitoring in CI?

Reply:
It is good to keep it this way.

There was change in one structure, so recompiling is needed.

Reply:
ah, yes then, libabigail should show that but might need to enable additional options for it to show when it's unsure

Reply:
> If SONAME is changing every time, we still have to rebuild even if ABI is the same, as SONAME is how things indicate ABI compatibility 

Fair point, it did update.


> but might need to enable additional options

I just used `qa-cmp` with libabigail installed :)